PublicMapping/districtbuilder
# Create tooltip component

## Overview

Adds wrapper component for [`react-tippy`](https://github.com/tvkhoa/react-tippy/) library and replaces `title` tags with new tooltip format.

### Demo

![image](https://user-images.githubusercontent.com/1809908/90281799-3c0ad100-de3b-11ea-985e-b0d0a3f00d41.png)

### Notes

- Replaced some class-based styling with ThemeUI-friendly `sx` props
- Fixed several of the svg icons, which had improper viewBox values

## Testing Instructions

- Open application
- Hover over race chart in sidebar
- Hover over the hand/rectangle or geounit switching buttons
- All of these should open tooltips

Closes #265
